packages/elements-react/src/theme/default/components/generic/page-header.tsx

```diff
@@ -3,7 +3,7 @@
 
 import { OryPageHeaderProps, useComponents } from "@ory/elements-react"
 import { UserMenu } from "../ui/user-menu"
-import { useSession } from "@ory/elements-react/client"
+import { useSession } from "../../../../client"
```

Comment:
revert, otherwise the theme bundle will contain a duplicate of `useSession` again, which leads to all sorts of issues.